docs/topics/settings.txt

```diff
@@ -46,7 +46,7 @@ The value of :envvar:`DJANGO_SETTINGS_MODULE` should be in Python path syntax,
 e.g. ``mysite.settings``. Note that the settings module should be on the
 Python `import search path`_.
 
-.. _import search path: https://diveinto.org/python3/your-first-python-program.html#importsearchpath
+.. _import search path: https://docs.python.org/3/library/sys_path_init.html#sys-path-init
```

Comment:
This is an incorrect anchor. Also, links to Python's documentation should use `:py:` directive.